.github/workflows/tests.yml

```
@@ -24,6 +24,7 @@ jobs:
       PGHOST: postgres
       PGUSER: discourse
       PGPASSWORD: discourse
+      USES_PARALLEL_DATABASES: ${{ matrix.build_type=='backend' && matrix.target=='core' }}
```

```suggestion
      USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' }}
```

^ that's the version we agreed on in the end, right? 😃

---

But if we make that change, the environment variable will be lying. Plugin backend tests do not use parallel databases.

I agree that we don't technically **need** the "target = core" check right now (because of the plugin migration hashes). But what if we decide to add parallel mode to backend plugin specs in future? I'd prefer the variable to describe the true situation.

---

(Also note that I used `USES_PARALLEL_DATABASES` to control the "create and migrate parallel databases" step. That change means we need the target==core check, otherwise we'll be migrating the parallel databases for no reason)

---

Ah, I thought all backend specs were using parallel dbs. 🙂 In this case, just the whitespace:

```suggestion
      USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' && matrix.target == 'core' }}
```